Cargo.toml

```diff
@@ -10,18 +10,15 @@ build = "build.rs"
 [[bin]]
 doc = false
 name = "miri"
-test = false
 path = "miri/bin/miri.rs"
 
 [[bin]]
 doc = false
 name = "cargo-miri"
-test = false
 path = "miri/bin/cargo-miri.rs"
 required-features = ["cargo_miri"]
 
 [lib]
-test = false
```

Comment:
iirc I added these because not having them recompiles miri with `cfg(test)` even though we know beforehand that this doesn't do anything.

Reply:
This had me confused when I added the first unittest in the lib (for the range map).

Reply:
ok. lets just leave it on. It's not that much with `CARGO_INCREMENTAL` on.